Cesium3DTilesSelection/src/TilesetJsonLoader.cpp

```diff
@@ -34,12 +36,17 @@
 #include <CesiumUtility/Assert.h>
 #include <CesiumUtility/ErrorList.h>
 #include <CesiumUtility/JsonHelpers.h>
+#include <CesiumUtility/Math.h>
 #include <CesiumUtility/Uri.h>
 
 #include <fmt/format.h>
 #include <glm/common.hpp>
+#include <glm/detail/setup.hpp>
```

Comment:
It's weird to need a detail header. Is this something clang-tidy wanted you to add?

Reply:
I _think_ clang-tidy wanted me to add this, but I could be wrong. I'll see if I can get away with removing it, or if there's a better file to include.